proto/Cargo.toml

```diff
@@ -12,19 +12,24 @@ description = "Public proto files for grpcio."
 categories = ["network-programming"]
 build = "build.rs"
 
+[features]
+default = ["protobuf-codec"]
+protobuf-codec = ["grpcio/protobuf-codec", "protobuf-codegen", "protobuf", "grpcio-compiler/protobuf-codec"]
+prost-codec = ["prost-derive", "bytes", "lazy_static", "grpcio/prost-codec", "prost-types", "prost", "grpcio-compiler/prost-codec"]
+
 [dependencies]
 futures = "0.1"
 grpcio = { path = "..", features = ["prost-codec"] }
```

Comment:
So by default, when depends on grpcio-proto, both prost-codec and protobuf-codec will be enabled?

Reply:
whoops, I missed that one. Addressed in the latest push